src/behavior.scm

```diff
@@ -23,10 +23,10 @@
 ; use of.
 ;
 ; A general OpenPsi rule has the form of if(context) then take(action);
-; these can contain variables, and can also be classed into different
+; these can contain variables, adn can also be classed into different
 ; groups based on the demands that they are fulfilling.
 ;
-; The content below consits entirely of actions to be taken; the
+; The content below consits entirely of actions to nbe taken; the
```

Comment:
You are adding another typo to the file.  Please review your changes BEFORE making a pull request.   It is even better to review your changes before making a commit, so that none of your commits introduce errors/bugs.  You can do this by saying "git diff" BEFORE you say "git commit", and then reviewing to make sure that everything looks OK.  Using "git diff"  is an excellent way to prevent bugs from getting introduced into the system.   The earlier you find a bug, the cheaper, easier and faster it is to fix the bug.   It just requires fewer people, and takes less time and effort (just think of the extra time and effort I just spent, just now, and think of the extra time and effort you will spend, fixing this.  It would have been easier to get this right the very first day, within a few minutes of the original "git commit")